test/controllers/v1/profiles_controller_test.rb

```diff
@@ -135,7 +135,10 @@ class ProfilesAuthenticationTest < ActionDispatch::IntegrationTest
           {
             'identity': {
               'account_number': account.account_number,
-              'auth_type': IdentityHeader::CERT_AUTH
+              'auth_type': IdentityHeader::CERT_AUTH,
+              'user': {
+                'is_internal': false
+              }
```

Comment:
Cert auth headers do not contain a 'user' within their 'identity'. They instead contain a 'system'; it will not include an 'is_internal' field. I pasted a link to a google doc in this jira that further explains the identity.